org.brlcad.BRL_CAD.yaml

```diff
@@ -0,0 +1,118 @@
+        x-checker-data:
+          type: anitya
+          project-id: 227581
+          stable-only: true
+          url-template: https://github.com/oneapi-src/oneTBB/archive/refs/tags/v$version.tar.gz
+
+  - name: blosc
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/Blosc/c-blosc.git
+        tag: v1.21.4
+        x-checker-data:
+          type: git
+          tag-pattern: ^(v[\d.]+)$
+
+  - name: openvdb
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/AcademySoftwareFoundation/openvdb.git
+        tag: v10.0.1
+        x-checker-data:
+          type: git
+          tag-pattern: ^(v[\d.]+)$
+
+  - name: BRL-CAD
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      cflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
+        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized -Wno-error=alloc-size-larger-than=
+      cxxflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
+        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized -Wno-error=alloc-size-larger-than=
+    config-opts:
+      - -DBRLCAD_BUNDLED_LIBS=ON
+      - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DBRLCAD_EXTRADOCS:BOOL=OFF
+      - -DBUILD_TESTING:BOOL=OFF
+      - -DBRLCAD_OPTIMIZED:BOOL=ON
+      - -DBRLCAD_ENABLE_QT:BOOL=ON
+      - -DBRLCAD_ENABLE_OPENVDB:BOOL=ON
+    post-install:
+      - install -Dm0644 ../${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm0644 ../misc/debian/mged.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm0644 ../misc/debian/rtwizard.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.rtwizard.desktop
+      - install -Dm0644 ../misc/debian/archer.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.archer.desktop
+      - install -Dm0644 ../misc/debian/brlcad-db.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.brlcad-db.desktop
+
+      - install -Dm0644 ../misc/debian/icons/256x256/mged.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm0644 ../misc/debian/icons/256x256/rtwizard.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.rtwizard.png
+      - install -Dm0644 ../misc/debian/icons/256x256/brlcad-db.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.brlcad-db.png
+      - install -Dm0644 ../misc/debian/icons/256x256/archer.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.archer.png
+
+      - desktop-file-edit --set-key=Categories --set-value="Graphics;Science;Education;Engineering;" ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop # Replace "BRL-CAD" category
```

Comment:
you should submit a patch upstream for these

Reply:
sure thing